grisera_api/registered_channel/registered_channel_service.py

```diff
@@ -1,18 +1,23 @@
 from graph_api_service import GraphApiService
-from registered_channel.registered_channel_model import RegisteredChannelIn, RegisteredChannelOut
 from channel.channel_service import ChannelService
+from registered_data.registered_data_service import RegisteredDataService
+from registered_channel.registered_channel_model import BasicRegisteredChannelOut, RegisteredChannelsOut, \
+    RegisteredChannelOut, RegisteredChannelIn
+from models.not_found_model import NotFoundByIdModel
+from models.relation_information_model import RelationInformation
 
 
 class RegisteredChannelService:
     """
     Object to handle logic of registered channels requests
 
     Attributes:
-        graph_api_service (GraphApiService): Service used to communicate with Graph API
-        channel_service (ChannelService): Service to send channel requests
+    graph_api_service (GraphApiService): Service used to communicate with Graph API
+    channel_service (ChannelService): Service to send channel requests
```

Comment:
registered_data_service (RegisteredDataService): Service to send registered data requests
😆